aliases.json

```diff
@@ -359,6 +359,24 @@
       }
     ]
   },
+  "gitter": {
+    "raw": [
+      {
+        "alias": "gitter-raw"
+      }
+    ],
+    "enrich": [
+      {
+        "alias": "gitter"
+      },
+      {
+        "alias": "alias_1"
```

Comment:
please add the aliases below and remove alias_1 and alias_2, thanks!
```
{
        "alias": "affiliations"
      },
      {
        "alias": "all_enriched"
      }
```

Reply:
Sorry I forgot to change that. Done now.